client/js/socket-events/more.js

```diff
@@ -14,7 +14,7 @@ socket.on("more", function(data) {
 
 	// get the scrollable wrapper around messages
 	const scrollable = chan.closest(".chat");
-	const heightOld = chan.height();
+	const heightOld = chan.height() - scrollable.scrollTop();
```

Comment:
This fixes scroll position not resetting correctly if we are not scrolled all the way up. This bug can be reproduced manually on master by scrolling a little so that load more button is clickable but not fully visible.